### PR TITLE
Get badger badge icons

### PR DESCRIPTION
**Pull request by termaatFlipside**

### CODE REVIEW

File: app/routes/app/$mType/m/$laborMarketAddress/prereqs.tsx [hunk 1]
```diff
@@ -46,8 +53,8 @@ export default function MarketplaceIdPrerequesites() {
                   You must hold this badge to review and score submissions on challenges
                 </p>
                 <div className="text-xs text-gray-500">MDAO S4 REVIEWER BADGE</div>
-                <div className="flex gap-2">
-                  <Avatar />
+                <div className="flex gap-2 items-center">
+                  <img src={maintainerURL} alt="" className="h-4 w-4" />
```
Comment (sebruiz) [inline app/routes/app/$mType/m/$laborMarketAddress/prereqs.tsx:57]: I _think_ we want to use Avatar for these (even those these might look more natural in rectangular form

File: app/routes/app/$mType/m/$laborMarketAddress/prereqs.tsx [hunk 2]
```diff
@@ -12,6 +12,13 @@ export default function MarketplaceIdPrerequesites() {
     throw new Error("MarketplaceIdPrerequesites must be rendered under a MarketplaceId route");
   }
   const { laborMarket } = data;
+  if (!laborMarket) {
```
Comment (sebruiz) [inline app/routes/app/$mType/m/$laborMarketAddress/prereqs.tsx:15]: can use `invariant` here to one line this

Comment (termaatFlipside) [reply app/routes/app/$mType/m/$laborMarketAddress/prereqs.tsx:15]: Gives errors about labor market being possibly null

Comment (termaatFlipside) [reply app/routes/app/$mType/m/$laborMarketAddress/prereqs.tsx:15]: nvm

**Review by sebruiz — APPROVED** [review]

**Pull request merged**